gps_waypoints_lat: [ 33.47030, 33.47016  ]
gps_waypoints_lon: [ -88.78603, -88.78555 ]